lib/sikuli/facebook.sikuli/facebook.py

```diff
@@ -427,17 +427,17 @@ def click_post_area_home(self, type='center'):
         @param type:
         @return:
         """
-        location = None
+        location = pattern = obj = None
```

Comment:
I think this will work fine.

However, it is somehow dangerous when it is not assigning `None`.
See `https://stackoverflow.com/questions/16348815/python-assigning-multiple-variables-to-same-value-list-behavior/16349356#16349356`
Maybe try ... `location, pattern, obj = [None, None, None]`?

@askeing thoughts?